casanovo/denovo/model_runner.py

```diff
@@ -255,7 +262,36 @@ def initialize_trainer(self, train: bool) -> None:
                 strategy=self._get_strategy(),
                 val_check_interval=self.config.val_check_interval,
                 check_val_every_n_epoch=None,
+                log_every_n_steps=1,
             )
+
+            if self.config._user_config.get("log_metrics"):
+                if not self.output_dir:
+                    logger.warning(
+                        "Output directory not set in model runner. "
+                        "No loss file will be created."
+                    )
+                else:
+                    CSV_LOG_SUB_DIR = "csv_logs"
```

Comment:
**question:** Not sure why this is all caps.

Reply:
It's a constant string I use more than once, but since it's not defined at the module level I'll go ahead and rename it.